src/web/mjs/connectors/MangaHost.mjs

```diff
@@ -49,7 +49,11 @@ export default class MangaHost extends Connector {
     }
 
     _getChapterList(manga, callback) {
-        this.fetchDOM(this.url + manga.id, 'div.chapters div.card.pop div.pop-title span')
+        this.fetchDOM(
+            // https://regex101.com/r/2gZjlK/1
+            manga.id.match(/(https?:)?(\/\/|\/\\\\)/) ?
```

Comment:
Looks complicated, can `const url = new URL(manga.id, this.url).href` not be used instead?

Reply:
ya but according to dodieboy's comment https://github.com/manga-download/hakuneko/issues/3362#issuecomment-823093119
so if the domain isn't the same, the URL constructor will return an exception...
I didn't just change the domain because probably the owner of mangahost is watching this project, whenever here is changed, there too

The only problem is that the clipboard will only work with the domain of this.url

Reply:
The URL constructor will automatically take care of it by providing a base URL
```javascript
new URL('/manga/one-piece', 'https://connector.domain').href;
// => 'https://connector.domain/manga/one-piece'
new URL('https://foreign.domain/manga/one-piece', 'https://connector.domain').href;
// => 'https://foreign.domain/manga/one-piece'
```

Reply:
For regularly changing domains, simply apply the new domain during initialization, e.g. as in [HeavenManga](https://github.com/manga-download/hakuneko/blob/master/src/web/mjs/connectors/HeavenManga.mjs#L20-L21)

Reply:
To support multi-domain copy & paste, overwrite the [canHandle()](https://github.com/manga-download/hakuneko/blob/master/src/web/mjs/connectors/HeavenManga.mjs#L13-L15) method with a corresponding pattern matching

Reply:
o✌️ 
I didn't know about multi-domain 👀